lib/audited/auditor.rb

```diff
@@ -55,9 +55,11 @@ def audited(options = {})
         Audit.audited_class_names << to_s
 
         on = Array(options[:on])
-        after_create :audit_create    if on.empty? || on.include?(:create)
-        before_update :audit_update   if on.empty? || on.include?(:update)
-        before_destroy :audit_destroy if on.empty? || on.include?(:destroy)
+        if_condition = options.delete(:if) { true }
```

Comment:
I think we might want to have an `unless` option as well, since in our case, we want auditing enabled unless some environment variable is set.  Otherwise, we'd need something like:

```ruby
audited only: :data, on: :update, if: ENV['AUDIT']
```

vesus

```ruby
audited only: :data, on: :update, unless: ENV['DISABLE_AUDIT']
```

This is really just a safeguard to protect us in case auditing causes us an issue.  We want it to default to enabled, unless we specify otherwise with an ENV variable.  So if we don't provide this ENV var, it will be enabled.